client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -691,6 +692,7 @@ export default function CompositeCheckout( {
 			<QuerySitePlans siteId={ siteId } />
 			<QuerySitePurchases siteId={ siteId } />
 			<QueryPlans />
+			<QueryExperiments />
```

Comment:
Ah! @niranjan-uma-shankar something not taken account of here is that experiments have a loading state.
While an experiment is loading neither control or treatment should be rendered.
Does this query happen early enough that a loading state can be safely ignored?

Reply:
There is an `isLoading` selector if need be.

Reply:
@jessie-ross The query happens early so the loading state can be safely ignored. This query happens when the checkout page loads, and only after the user has completed purchase of a Personal plan do we determine to show the offer page based on the variation.